Chrome & FireFox/injection-scripts/universal-script.js

```diff
@@ -0,0 +1,121 @@
+// Makes the background dark avoid a white flash at load.
+let rootstyle = document.querySelector(':root').style;
+rootstyle.setProperty('--color-background-primary', '#181A1B');
+
+// Creates the link-elements.
+let specificStyle = document.createElement('link');
+specificStyle.id = 'EcosiaDarkModeSpecific';
+specificStyle.className = 'EcosiaDarkMode';
+specificStyle.rel = 'stylesheet';
+
+let universalStyle = document.createElement('link');
+universalStyle.id = 'EcosiaDarkModeUniversal';
+universalStyle.className = 'EcosiaDarkMode';
+universalStyle.rel = 'stylesheet';
+universalStyle.href = chrome.runtime.getURL('injection-styling/universal-styling.css');
+
+let styles = [universalStyle];
+
+// Initial injection - gets the settings and applies them.
+chrome.storage.local.get(['settings'], injectOnLoad);
+
+// Sets a timeout to the next minute-change.
+setTimeout(() => {
+    // Runs it one time first because setInterval has to wait for one minute before it can start.
+    intervalcheck();
+
+    // Checks the time every minute.
+    setInterval(() => {
+        intervalcheck();
+    }, 60000);
+}, (60 - (new Date().getSeconds())) * 1000);
+
+// Subscribe to other necessary events.
+document.addEventListener('DOMContentLoaded', changeStyleImportance, false);
+chrome.runtime.onMessage.addListener(updateStyle);
+
+
+
+// Inject the style into the document.
+function inject(styles) {
+    styles.forEach(style => {
+        (document.body || document.head || document.documentElement).appendChild(style);
+    });
+    setTimeout(() => {
+        document.querySelector(':root').style = '';
+    }, 200);
```

Comment:
Quick question, what's this timeout for?

Reply:
It removes the style that is applied to the background to remove the flash. I think I ended up with small flashes from time to time because it removed it too early, so I added the timeout.